src/Schema/Values/OwnerType.ts

```diff
@@ -33,6 +33,7 @@ export enum OwnerType {
   collect = "collect",
   collection = "collection",
   collections = "collections",
+  collectionsCategory = "collectionsCategory",
```

Comment:
This is very easy to miss (and annoying to discover later) but this value won't be easily accessible by Eigen unless it's also included in the ScreenOwnerType definition below ~L158.

Reply:
oh good catch, I think we can enhance these typings(ScreenOwnerType) in the future with Omit and Pick to make it lighter for engineers working with it

Reply:
Yeah, for engineers that truly want to work around the typing they can always force it with `"somethingElse" as ScreenOwnerType` but at least that's made explicit in the downstream code.